sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -336,6 +373,7 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 <!-- LINKS -->
+[aad_authorization]: https://docs.microsoft.com/en-us/azure/cognitive-services/authentication#authenticate-with-azure-active-directory
```

Comment:
it will be good to delete `en-us` from all the links to docs.microsoft so that it resolves according to user location